esmvalcore/preprocessor/_io.py

```diff
@@ -461,3 +470,84 @@ def _concatenate_overlapping_cubes(cubes):
                 raise ex
 
     return cubes
+
+def _equalise_attributes(cubes_list):
```

Comment:
there is already an `iris` function available, funnily enough named the same, [`equalise_attributes`](https://scitools-iris.readthedocs.io/en/latest/generated/api/iris/util.html#iris.util.equalise_attributes)